common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -184,6 +178,33 @@ protected SingleThreadEventExecutor(EventExecutorGroup parent, Executor executor
         rejectedExecutionHandler = ObjectUtil.checkNotNull(rejectedHandler, "rejectedHandler");
     }
 
+    /**
+     * Called from arbitrary non-eventloop thread prior to scheduled task submission.
```

Comment:
`non-eventloop` -> `non {@link EventExecutor}`

Reply:
Should we also rename `SingleThreadEventExecutor#inEventLoop()`? :trollface: